.github/actions/integration-tests/run-ci-stage1

```diff
@@ -618,6 +618,8 @@ run_cmd "crucible ls --type run-id"
 
 run_cmd "crucible ls --type tags"
 
+run_cmd "crucible opensearch repair"
```

Comment:
wait... did I miss anything on this PR? this has nothing to do with the proposed patch

Reply:
It's a separate commit.  I had both things to do so I lumped them together, primarily because running CI for `crucible-ci` takes so long (only 740 jobs now I think!!!).